snowfakery/__init__.py

```diff
@@ -10,3 +11,35 @@
 version_file = Path(__file__).parent / "version.txt"
 with version_file.open() as f:
     version = f.read().strip()
+
+
+def generate_data(
```

Comment:
This is a lot of args. Should we add `*` to force the caller to use named keyword arguments, so that it's not possible to use positional args which could be unclear and would make it hard for us to change the order?

Reply:
Good idea.